commands/sense/README.md

```diff
@@ -62,4 +63,24 @@ You can add more meta about the extension by providing a `.json` formatted file
 nebula sense --meta meta.json
 ```
 
+The meta data will be ended up in the `.qext` file used by the Qlik Sense.
+
 The rest of the required information is populated automatically based on the content in `package.json`.
+
+### Output
+
+Generate all required files into the specified `--output` folder called sn-table-ext:
+
+```bash
+nebula sense --output sn-table-ext
+```
+
+You can upload that folder as an extension on [Qlik Sense Enterprise for Windows](https://help.qlik.com/en-US/sense-developer/April2020/Subsystems/Extensions/Content/Sense_Extensions/Howtos/deploy-extensions.htm) or [Qlik Sense Enterprise on Kubernetes](https://help.qlik.com/en-US/sense-admin/April2020/Subsystems/DeployAdministerQSE/Content/Sense_DeployAdminister/QSEoK/Administer_QSEoK/mc-extensions.htm)
```

Comment:
Should update these links to the latest help.
Also switch from "on Kubernetes" to "Saas editions of Qlik Sense" https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Admin/mc-extensions.htm